### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by dwagmuse**
documentation

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,2 +1,18 @@
 # md-adapter
-The MagicDraw adapter framework for OML
+
+The MagicDraw (Cameo System Modeler) adapter framework for OML
+
+This framework consists of two major parts:
+
+* Profile Generator
+The profile generator converts a set of OML vocabularies into a SysML profile that can be installed in Magicdraw enabling the modeler to use the source vocabulary concepts and relations in a SysML model to create models with precise semantics.
```
Comment (melaasar) [inline README.md:8]: Replace SysML by UML

File: README.md [hunk 2]
```diff
@@ -1,2 +1,18 @@
...
+The input vocabulary is given in two parts: the source vocabulary and an associated embedding vocabulary that establishes the relationship between each element of the source vocabulary and a target SysML element it will become a stereotype for.
+
+* Export Adapter
+The export adapter will transform SysML models that apply profiles generated using the profile generator into OML. The exported OML models can be subsequently transformed to OWL and given to a reasoner to verify the consistency of the instance model with the vocabulary. The OWL data can also be stored in a graph database to make it accessible to queries.
+
+These elements are not yet implemented in openCAESAR. The legacy verions are available in our legacy repo. The legacy versions use a legacy version of the OML language (OML-1).
```
Comment (melaasar) [inline README.md:14]: Avoid saying  "our" (legacy repo).

File: README.md (hunk 1 above)
Comment (dwagmuse) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (dwagmuse) [reply README.md]: done

**Pull request merged**